### PR TITLE
Fix compatibility with NPM

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,28 +13,22 @@
...
     "@babel/eslint-parser": "^7.18.9",
     "@vue/cli-plugin-babel": "~5.0.8",
     "@vue/cli-plugin-e2e-cypress": "~5.0.8",
     "@vue/cli-plugin-eslint": "~5.0.8",
     "@vue/cli-plugin-router": "~5.0.8",
     "@vue/cli-plugin-unit-jest": "~5.0.8",
     "@vue/cli-service": "~5.0.8",
-    "@vue/compiler-sfc": "^3.2.37",
-    "@vue/eslint-config-standard": "^8.0.0",
-    "@vue/eslint-plugin": "^4.2.0",
```
Comment (davidyuk) [inline package.json:30]: This package was causing instalation issue, it is 5 years old, was used before @vue/cli

File: package.json [hunk 2]
```diff
@@ -13,28 +13,22 @@
...
+    "@babel/core": "^7.18.9",
     "@babel/eslint-parser": "^7.18.9",
     "@vue/cli-plugin-babel": "~5.0.8",
     "@vue/cli-plugin-e2e-cypress": "~5.0.8",
     "@vue/cli-plugin-eslint": "~5.0.8",
     "@vue/cli-plugin-router": "~5.0.8",
     "@vue/cli-plugin-unit-jest": "~5.0.8",
     "@vue/cli-service": "~5.0.8",
-    "@vue/compiler-sfc": "^3.2.37",
-    "@vue/eslint-config-standard": "^8.0.0",
```
Comment (davidyuk) [inline package.json:29]: This config wasn't enabled, and I removed it completely, keeping only rules for error prevention.

File: package.json [hunk 3]
```diff
@@ -48,9 +42,8 @@
   },
   "license": "ISC",
   "lint-staged": {
-    "*.{js,jsx,vue,ts,tsx}": [
-      "vue-cli-service lint",
-      "git add"
```
Comment (davidyuk) [inline package.json:53]: Vue cli generate this part without this line, so it looks like it can be omitted 🤔

**Comment by marc0olo** [comment]
@davidyuk I was shortly playing around with fixing tests etc. but this is a mess for me. can you check the tests and maybe fix or delete the unnecessary and unused stuff, too? 😅

**Review by thepiwo — APPROVED** [review]

**Review by thepiwo — CHANGES_REQUESTED** [review]
readme still states yarn usage

**Comment by thepiwo** [comment]
replaced by this one https://github.com/aeternity/aepp-boilerplate-vue/pull/18

**Pull request closed without merging**